clusterloader2/api/validation/validation_test.go

```diff
@@ -260,8 +261,8 @@ func TestVerifyStep(t *testing.T) {
 		},
 		{
 			name: "has measurements and no phases",
-			input: Step{
-				Measurements: []*Measurement{
+			input: api.Step{
+				Measurements: []*api.Measurement{
 					{
 						Method: "test1",
```

Comment:
@marseel here is the dummy measurement!

Reply:
You can replace "test1" with measurement that actually exists and it will work.

Reply:
@marseel , every measurement needs to be regsitered,right? so lets say here if i use 
```
Measurements: []*api.Measurement{
					{
						Method: "APIAvailability",
					},
				},
```

still it will say its not registered! As we did in our new test case first register , should we also do same in this case?

Reply:
/cc @marseel